js/views/itemEditVw.js

```diff
@@ -84,6 +88,20 @@ module.exports = Backbone.View.extend({
           }
       });
 
+      editor.subscribe('blur', function(e) {
+        var $field = self.$('#inputDescription');
+
+        if (!$($field.val()).text().length) {
+          $field.val('');
+        }
+
+        if (!$field[0].checkValidity()) {
+          $field.parent().addClass('invalid');
+        } else {
+          $field.parent().removeClass('invalid');
+        }
+      });
+
```

Comment:
For this field that won't fire because the MediumEditor hijacks the field and consequently the standard dom blur doesn't fire. Also, it's pointless to make the textarea have an :invalid state, because it's hidden, so the styling will never show, which is why I add an 'invalid' class to the parent.

If there are no other textarea's in the view, we could probably remove the backbone event declaration.